README.md

```diff
@@ -12,7 +12,7 @@ Install the package via composer by adding this section to the composer.json fil
 
 ```JSON
 "require": {
-    "revinate/sequence": "~0.4"
+    "jorgezf/sequence": "1.0.0"
 },
 ```
```

Comment:
I think that instead we should indicate how to install it from command line:

```markdown

\```bash
composer require jorgezf/sequence
\```

```

Reply:
Updated d6000af6e2fdac83758117eaec5409b1169c1968